### PR TITLE
NEXT-8427 - Rename EAN to GTIN

### LINKED ISSUE #803: NEXT-8427 - EAN rename to GTIN
EN:
For the EAN, the obsolete designation is used ("European Article Number") which has been replaced by GTIN ("Global Trade Item Number") since 2005. We should adapt this at least in the English language.

DE:
Für die EAN wird die veraltete Bezeichnung verwendet ("European Article Number") die seit 2005 durch GTIN ("Global Trade Item Number") abgelöst wurde. Dies sollten wir mindestens in der englischen Sprache anpassen.

### PR DESCRIPTION
**Pull request by ngoedde**
fixed #803 

### 1. Why is this change necessary?
For the EAN, the obsolete designation is used ("European Article Number") which has been replaced by GTIN ("Global Trade Item Number") since 2005. We should adapt this at least in the English language.

### 2. What does this change do, exactly?
This changes `EAN` to `GTIN` in the administration and adds a new label `GTIN` to the product detail page (if a GTIN is available for this product).

This PR does not change the field names inside the database or product definition to decrease the risk of bugs.

Since none of the actual logic was touched, a test is not required.

### 3. Describe each step to reproduce the issue or behaviour.
Set the GTIN of any product in the administration and go to the detail page. You should now be able to see a new Label "GTIN" below the product order number label. 

### 4. Please link to the relevant issues (if any).
https://github.com/shopware/platform/issues/803

### 5. Checklist

- [X] I have written tests and verified that they fail without my change
No tests required
- [X] I have squashed any insignificant commits
- [X] I have written or adjusted the documentation according to my changes
- [X] This change has comments for package types, values, functions, and non-obvious lines of code
- [X] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: CHANGELOG-6.3.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ To get the diff between two versions, go to https://github.com/shopware/platform
         * Added `sw-sales-channel-google-merchant` component to show existing merchant accounts list and handle assigning existing merchant account or creating new account
         * Added `sw-sales-channel-google-shipping-setting` component to handle shipping setting selection
     * Added salesChannel state in `sw-sales-channel` module
+    * Rename snippet `EAN` to `GTIN`
 
 * Core    
 
 * Storefront
-
+    * Added `GTIN` label to the detail page
```
Comment (mitelg) [inline CHANGELOG-6.3.md]: please do not use the 6.3 changelog, as the boost day branch will become a 6.2.x release

**Comment by mitelg** [comment]
please also squash your commits :blush:

File: CHANGELOG-6.3.md (hunk 1 above)
Comment (ngoedde) [reply CHANGELOG-6.3.md]: Done... derp.

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json [hunk 2]
```diff
@@ -94,7 +94,7 @@
       "placeholderMaxPurchase": "Maximalabnahme ...",
       "placeholderPurchaseSteps": "Verkaufsintervall (Stückzahl) ...",
       "placeholderRestockTime": "Wiederauffüllintervall ...",
-      "placeholderEan": "Gib die EAN-Nummer ein ...",
+      "placeholderEan": "Gib die GTIN-Nummer ein ...",
```
Comment (tinect) [inline src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json]:
```suggestion
      "placeholderEan": "Gib die GTIN ein ...",
```
GTIN stays for `Global Trade Item **Number**`

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/en-GB.json [hunk 3]
```diff
@@ -93,7 +93,7 @@
       "placeholderMaxPurchase": "Max. amount purchaseable...",
       "placeholderPurchaseSteps": "Enter purchase steps...",
       "placeholderRestockTime": "Enter restock time in days...",
-      "placeholderEan": "Enter EAN number...",
+      "placeholderEan": "Enter GTIN number...",
```
Comment (tinect) [inline src/Administration/Resources/app/administration/src/module/sw-product/snippet/en-GB.json]:
```suggestion
      "placeholderEan": "Enter GTIN...",
```
GTIN stays for `Global Trade Item **Number**`

**Comment by GitEvil** [comment]
QA approved :heavy_check_mark:

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json (hunk 2 above)
Comment (ngoedde) [reply src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json]: Well.. good point! I changed it.

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/en-GB.json (hunk 3 above)
Comment (ngoedde) [reply src/Administration/Resources/app/administration/src/module/sw-product/snippet/en-GB.json]: Done

**Comment by pantrtxp** [comment]
@ngoedde can you please resolve your conflicts?

**Comment by mitelg** [comment]
@ngoedde please do a rebase and not a merge commit :wink:

**Comment by ssltg** [comment]
@ngoedde can you please do a rebase? Thank you

**Comment by lernhart** [comment]
Due to 14 days inactivity on this, I will close the PR now.

**Pull request closed without merging**